Tests/MAPL3G_Component_Testing_Framework/test_cases/case18/GCM1.yaml

```diff
@@ -15,23 +15,10 @@ mapl:
 
   states:
     export:
-      #E_1:
-        #standard_name: 'foo'
-        ##class: vector
-        ##standard_name: '(eastward_wind, northward_wind) horizontal velocity'
-        ##vector_component_names: [U, V]
-        #units: "m s-1"
-        #typekind: R4
-        #default_value: 17.
-        #vertical_dim_spec: NONE
-      U:
-        standard_name: 'eastward_wind'
-        units: "m s-1"
-        typekind: R4
-        default_value: 17.
-        vertical_dim_spec: NONE
-      V:
-        standard_name: 'northward_wind'
+      UV:
+        class: vector
+        standard_name: '(eastward_wind, northward_wind) horizontal velocity'
+        vector_component_names: [U, V]
```

Comment:
We may want to make a shorter alias for this key.  Not very user friendly, despite the clarity it provides.  (Not in this PR.  It is fodder for a review of the entire MAPL3 API.)